opensearchpy/helpers/actions.py

```diff
@@ -586,14 +586,17 @@ def scan(
     scroll_id = resp.get("_scroll_id")
 
     try:
-        while scroll_id and resp["hits"]["hits"]:
-            for hit in resp["hits"]["hits"]:
+        while scroll_id and resp.get("hits", {}).get("hits"):
```

Comment:
Hi @Djcarrillo6, I've reviewed the code. Thanks for your contribution! Could you please make similar adjustments in the async helpers as well?

Reply:
@saimedhi absolutely!

Reply:
You should be able to run `nox -rs generate` and it will make all those changes for you I believe.